src/towncrier/build.py

```diff
@@ -26,6 +28,18 @@ def _get_date() -> str:
     return date.today().isoformat()
 
 
+def validate_answer(ctx: Context, param: Option, value: bool) -> bool:
```

Comment:
Maybe just make it a private method, not to encourage its usage outside of towncrier.

I thing that the missing docs is ok.
I can underestand the code even without docs.

```suggestion
def _validate_answer(ctx: Context, param: Option, value: bool) -> bool:
```